ic-os/docs/Configuration.adoc

```diff
@@ -29,18 +36,14 @@ GuestOS then reads from `ic-bootstrap.tar` (link:../components/init/bootstrap-ic
 * *System state:* Node initialization material (initial registry & keys) is copied directly into target directories for replica use.
 * *Config partition:* All other contents of the injected config are copied into the `/boot/config` directory. These configurations need to be available on subsequent boots (not just the first boot of the system), and need to be preserved across upgrades.
 ** The contents of `/boot/config` are then used in various ways to parameterize system services as they start. For example, ssh keys are installed to target accounts, network settings are applied by network set-up scripts, etc.
-** For the IC, the `generate-replica-config.sh` script uses the `ic.json5.template` file to produce the configuration required by the replica (also cf. service file definition). Template placeholders are populated with information from `/boot/config` and elsewhere.
+** `generate-replica-config.sh` uses `ic.json5.template` to produce the replica configuration (also cf. service file definition). Template placeholders are populated with information from `/boot/config` and elsewhere.
```

Comment:
> and elsewhere

😬

Reply:
I just double checked and from what I can see template placeholders are populated with information from `/boot/config` so I removed the 'and elsewhere'